e2e/internal/az/image.go

```diff
@@ -15,6 +16,10 @@ import (
 // NullImageVersion is the version returned when no image version is found.
 const NullImageVersion = "0.0.0"
 
+// oldImageVersionCodenames is a list of Ubuntu codenames for which Azure images
+// are built using the previous image versioning format.
+var oldImageVersionCodenames = []string{"focal", "jammy", "mantic"}
```

Comment:
nitpick: the new will become old at some port, but as they didn’t version the format… :p